plugins/woocommerce/client/admin/client/settings-payments/index.tsx

```diff
@@ -160,12 +166,26 @@ export const SettingsPaymentsMethodsWrapper = () => {
 				) }
 				hasButton={ true }
 				buttonLabel={ __( 'Continue', 'woocommerce' ) }
-				onButtonClick={ onButtonClick }
+				onButtonClick={ onClick }
 			/>
 			<Suspense
-				fallback={ <div>Loading payment methods settings...</div> }
+				fallback={
+					<>
+						<div className="settings-payments-recommended__container">
+							<div className="settings-payment-gateways">
+								<ListPlaceholder
+									rows={ 3 }
+									hasDragIcon={ false }
+								/>
+							</div>
+						</div>
+					</>
+				}
 			>
-				<SettingsPaymentsMethodsChunk />
+				<SettingsPaymentsMethodsChunk
+					paymentMethodsState={ paymentMethodsState }
```

Comment:
do we need to pass it in chunk? I thought it's not necessary

Reply:
Yes, the state must be passed down to the child components to ensure it can be utilized.